about/user-testing.md

```diff
@@ -0,0 +1,94 @@
+Join arXiv's User Testing group
+===========================
+
+<style>
+blockquote {
+  border-left: 0;
+  -webkit-box-shadow: 0px 3px 8px 0px rgba(0,0,0,0.12);
+  -moz-box-shadow: 0px 3px 8px 0px rgba(0,0,0,0.12);
+  box-shadow: 0px 3px 8px 0px rgba(0,0,0,0.12);
+  padding:1em;
+  margin-bottom:1.5em;
+}
+@media (min-width: 576px) {
+  blockquote {
+    padding: 2em;
+  }
+}
+</style>
+
+From time to time arXiv sends out surveys, previews to new changes, and other testing opportunities to interested users. If you would like to join our user testing group fill out the form below. User testing is integral to arXiv's development process and we welcome your feedback. We use this list only for user testing communications and never spam or share your information.
+
+_(note: we recently migrated our list from an old-fashioned listserv to mailchimp. The new platform lets us acknowledge important testing criteria like geographic location and use of assistive technology. If you are already part of the user testing list but would like to contribute to a targeted testing group simply fill out the required fields, hit submit, and follow the instructions to update your existing account)_
+
+
+<link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
+<style type="text/css">
+	#mc_embed_signup{background:#fff; clear:left; font-size:14px;}
+  #mc_embed_signup .mc-field-group label{font-style:normal;}
+  #mc_embed_signup .mc-field-group.input-group label{margin-left:5px;}
+  #mc_embed_signup .mc-field-group select {height:35px;}
+  @media only screen and (min-width: 600px) {
+    #mc_embed_signup{max-width:75%; margin:0 auto;}
+  }
+</style>
+> <div id="mc_embed_signup">
+> <h3>Sign up for user testing</h3>
+>
+>
+> <form action="https://arxiv.us4.list-manage.com/subscribe/post?u=31c4aaf571c921df1fb50adee&amp;id=7cd72795aa" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+<div id="mc_embed_signup_scroll">
+<div class="mc-field-group">
+<label for="mce-EMAIL">Email Address * </label>
+<input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL">
+</div>
+<div class="mc-field-group">
+<label for="mce-FNAME">First Name </label>
+<input type="text" value="" name="FNAME" class="" id="mce-FNAME">
+</div>
+<div class="mc-field-group">
+<label for="mce-LNAME">Last Name </label>
+<input type="text" value="" name="LNAME" class="" id="mce-LNAME">
+</div>
+<div class="mc-field-group">
+<label for="mce-MMERGE6">Country * </label>
+<input type="text" value="" name="MMERGE6" class="required" id="mce-MMERGE6">
+</div>
+<div class="mc-field-group">
+	<label for="mce-RESEARCH">arXiv Research Field * </label>
+	<select name="RESEARCH" class="required" id="mce-RESEARCH">
+	<option value=""></option>
+	<option value="astro-ph">astro-ph</option>
```

Comment:
Please add other physics major groups
physics, cond-mat, hep

Reply:
The list was getting a bit long so Ive instead updated the select options to match the Homepage main headings (Physics, Mathematics, etc). I think we only need broad strokes. @jimentwood does that sound reasonable?

Reply:
Yes, sounds good